apps/web/src/app/(dashboard)/documents/data-table-action-dropdown.tsx

```diff
@@ -55,14 +55,14 @@ export const DataTableActionDropdown = ({ row }: DataTableActionDropdownProps) =
   }
 
   const recipient = row.Recipient.find((recipient) => recipient.email === session.user.email);
-
   const isOwner = row.User.id === session.user.id;
+
   // const isRecipient = !!recipient;
   const isDraft = row.status === DocumentStatus.DRAFT;
   // const isPending = row.status === DocumentStatus.PENDING;
   const isComplete = row.status === DocumentStatus.COMPLETED;
   // const isSigned = recipient?.signingStatus === SigningStatus.SIGNED;
-  const isDocumentDeletable = isOwner;
+  const isDocumentDeletable = isOwner && !recipient;
```

Comment:
This isn't quite right, we just want to stop deletion if it's already been deleted and you're the owner. This is kind of redundant since you'll only retain the document if it's completed and so deletion won't do much but is a point to clarify.